code/modules/mining/machine_vending.dm

```diff
@@ -55,6 +55,8 @@
 		new /datum/data/mining_equipment("Drone Ranged Upgrade",/obj/item/device/mine_bot_ugprade/cooldown,								600),
 		new /datum/data/mining_equipment("Drone AI Upgrade",	/obj/item/slimepotion/sentience/mining,									1000),
 		new /datum/data/mining_equipment("Jump Boots",			/obj/item/clothing/shoes/bhop,											2500),
+		new /datum/data/mining_equipment("Anti-Ash Walker injector",	/obj/item/weapon/storage/box/scalerot/pen,						2000),
```

Comment:
None of the messages work for Ash Walkers?

Hm, I didn't know that. That's interesting. I need to find out why then.

Are Ashwalkers a completely seperate race?